### ISSUE TITLE
Please grant me write access to this repository so I can create release branches -- and then PR to those release branches

### ISSUE THREAD

**Issue by philCryoport** [asyncapi/jasyncapi-idea-plugin#10]
# Summary
The content below is basically the same as in [this AsyncAPI Slack post](https://asyncapi.slack.com/archives/C02FK3YDPCL/p1672958682043019).

@Pakisan & @derberg : per @fmvilas 's [suggestion in the AsyncAPI Slack](https://asyncapi.slack.com/archives/CQVJXFNQL/p1673280765233719?thread_ts=1672165038.512309&cid=CQVJXFNQL), I'm formally requesting `write` permission for this repository so that I can create release branches -- and then issue PRs from my own fork to those release branches.
(If I'm reading the GitHub docs correctly, `write` permission is required to create branches.)

I'm not looking to code-review until I get more familiar with this code-base. Towards that end, please separately [see my PR that makes the two of you CODEOWNERS](https://github.com/asyncapi/jasyncapi-idea-plugin/pull/9) so that you are the ones who approve of PRs.

# Why do I want to create release branches?
* I use this plugin heavily at the moment
* Understandably, I'd rather run off of a code-reviewed release version than a self-built one
* At the moment I need two release branches created so I can roll the revs and submit PRs:
  * one for IntelliJ IDEA 2022.2.4 -- as @Pakisan said he wanted to tackle that first before going to the 2022.3.x releases
  * one for IntelliJ IDEA 2022.3.1
  * ...and who knows when IntelliJ IDEA 2023.x will drop
  * ...plus once a PR for IntelliJ IDEA 2022.2.4 is approved, I have waiting [this PR for IntelliJ IDEA 2022.3](https://github.com/asyncapi/jasyncapi-idea-plugin/pull/8)

# About me
* I've contributed multiple PRs to this plugin, both [before](https://github.com/Pakisan/jasyncapi-idea-plugin/pull/47) and [after](https://github.com/asyncapi/jasyncapi-idea-plugin/pull/8) it became an AsyncAPI-owned repo.
* [My LinkedIn](https://linkedin.com/in/philiphschlesinger)
* [My personal Github](https://github.com/theschles)
* [My work Github](https://github.com/philCryoport/)

# In summary
Please consider this request to grant me `write` privileges for this repo so I can take care of rev rolls -- and then all the two of you need to do is review and approve the PRs.

Thank you!

**Comment by derberg** [comment]
@Pakisan, as I'm only helping here with logistics, it all depends on you as you contributed to this project. I know you were looking for help, we shared this "call for help" from official AsyncAPI accounts, and looks like it finally happened. Please have a look and lemme know how I can help.
